### PR TITLE
Fake pr

### PR DESCRIPTION
**Pull request by opotowsky**
## Description

---

## Checklist

- [ ] This PR has only one purpose or idea.
- [ ] Tests have been added/updated to verify that the new/changed code works.

- [ ] The code style follows [good practices](https://terrapower.github.io/armi/developer/standards_and_practices.html).
- [ ] The commit message(s) follow [good practices](https://terrapower.github.io/armi/developer/tooling.html).

- [ ] The [release notes](https://terrapower.github.io/armi/release/index.html) (location `doc/release/0.X.rst`) are up-to-date with any bug fixes or new features.
- [ ] The documentation is still up-to-date in the `doc` folder.
- [ ] The dependencies are still up-to-date in `setup.py`.

### CODE REVIEW

File: .github/workflows/pr-quality.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Enforce Pull Request Quality
 
-on: [pull_request]
+on: pull_request
```
Comment (john-science) [inline .github/workflows/pr-quality.yaml]: You could try replacing this line with something like:

```
on:
  push:
    branches:
      - main
  pull_request:
```

Comment (john-science) [reply .github/workflows/pr-quality.yaml]: Or, just for testing:

```
on:
  push:
    branches:
  pull_request:
```

**Comment by john-science** [comment]
Your `pr-quality.yaml` file references this file:

> configuration-path: .github/workflows/labeler.yaml

But I don't see that one in your repo.

Is that the issue?

**Comment by opotowsky** [comment]
> Your `pr-quality.yaml` file references this file:
> 
> > configuration-path: .github/workflows/labeler.yaml
> 
> But I don't see that one in your repo.
> 
> Is that the issue?

I'm actually PR-ing it into a branch with file already in it.

**Pull request closed without merging**